### ISSUE TITLE
Implement re-focus 

### ISSUE THREAD

**Issue by Irineu333** [Irineu333/SpeakTouch#70]
Just like **Talkback** and **Jieshou**, **SpeakTouch** should be able to re-read the focused view

**Note:** This feature cannot interfere with #66
